Branch-SDK/src/main/java/io/branch/coroutines/InstallReferrers.kt

```diff
@@ -258,5 +347,39 @@ fun getLatestValidReferrerStore(allReferrers: List<InstallReferrerResult?>): Ins
         it.latestInstallTimestamp
     }
 
+    if (allReferrers.filterNotNull().any { it.appStore == Jsonkey.Meta_Install_Referrer.key }) {
+        return handleMetaInstallReferrer(allReferrers, result!!)
+    }
+
     return result
+}
+
+//Handle the deduplication and click vs view logic for Meta install referrer
```

Comment:
@nsingh-branch don't we have to check all install referrers and not just Google? in this scenerio? cc: @gdeluna-branch

Reply:
@jf-branch In which part?

Reply:
For line 366, unless we do this for all others against Google as well, im not super familiar, but we can discuss live.

Reply:
I believe that Meta ads only redirect to the Google Play store, so the deduplication should only be required for the Play Store referrer